os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -116,10 +117,13 @@
 
 #define OVER_SAMPLE_RATE (384U)
 
-#define I2S_DMA_PAGE_SIZE	768   // 2 ~ 4096
-#define I2S_DMA_PAGE_NUM    4   // Vaild number is 2~4
+#define I2S_DMA_PAGE_SIZE	1024   /* 4 ~ 16384, set to a factor of APB size */
+#define I2S_DMA_PAGE_NUM    4   /* Vaild number is 2~4 */
 
 /* I2S buffer container */
+#ifdef __16_BIT_PADDING__
```

Comment:
above default values are acceptable, but regarding PADDING, we need a KConfig

Reply:
Noted, I have added a AMEBAD_I2S_16BIT_PADDING option in KConfig to turn on the feature